### ISSUE TITLE
Warning when BC checks for updates

### ISSUE THREAD

**Issue by JasonHorkles** [AuraDevelopmentTeam/BungeeChat2#97]
#### Issue Description: 
It's a warning

#### What happens:
I get this warning: `[21:46:34 WARN] [BungeeChat]: Could not fetch the latest version!
java.io.FileNotFoundException: https://www.spigotmc.org/api/general.php
        at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
        at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
        at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
        at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
        at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1950)
        at sun.net.www.protocol.http.HttpURLConnection$10.run(HttpURLConnection.java:1945)
        at java.security.AccessController.doPrivileged(Native Method)
        at sun.net.www.protocol.http.HttpURLConnection.getChainedException(HttpURLConnection.java:1944)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1514)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1498)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getInputStream(HttpsURLConnectionImpl.java:268)
        at dev.aura.bungeechat.BungeeChat.queryLatestVersion(BungeeChat.java:286)
        at dev.aura.bungeechat.BungeeChat.getLatestVersion(BungeeChat.java:312)
        at dev.aura.bungeechat.listener.VersionCheckerListener$VersionCheckerTask.run(VersionCheckerListener.java:45)
        at net.md_5.bungee.scheduler.BungeeTask.run(BungeeTask.java:63)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)
Caused by: java.io.FileNotFoundException: https://www.spigotmc.org/api/general.php
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream0(HttpURLConnection.java:1896)
        at sun.net.www.protocol.http.HttpURLConnection.getInputStream(HttpURLConnection.java:1498)
        at java.net.HttpURLConnection.getResponseCode(HttpURLConnection.java:480)
        at sun.net.www.protocol.https.HttpsURLConnectionImpl.getResponseCode(HttpsURLConnectionImpl.java:352)
        at dev.aura.bungeechat.BungeeChat.queryLatestVersion(BungeeChat.java:283)
        ... 6 more`

#### What you expected to happen:
No warning

#### Steps to reproduce:

1. Join the server with the right perms
2. See the warning in console
...

____
#### Affected Versions: 

- Minecraft: 1.15.1
- BungeeChat:  2.1.7.1067-DEV
- Other related plugins: 

In other words just use a different API link since the current one throws a 404 error

**Comment by BrainStone** [comment]
According to https://www.spigotmc.org/threads/update-checking.264850/ this URL has been deprecated for 2.5 years. Will update the check.
